plugins/woocommerce-blocks/assets/js/base/components/select/index.tsx

```diff
@@ -9,11 +9,17 @@ import { useCallback, useId } from '@wordpress/element';
  */
 import './style.scss';
 
+export type SelectOption = {
+	value: string;
+	label: string;
+	disabled?: boolean;
+};
+
 type SelectProps = Omit<
 	React.SelectHTMLAttributes< HTMLSelectElement >,
 	'onChange'
 > & {
-	options: { value: string; label: string }[];
+	options: { value: string; label: string; disabled?: boolean }[];
```

Comment:
can we change this to `SelectOption[]`?

```suggestion
	options: SelectOption[];
```

Reply:
Good spotting, thanks I'll fix this.

Reply:
Resolved in [9a36764](https://github.com/woocommerce/woocommerce/pull/49616/commits/9a367642a29a1d5b7f3c74f39469402bf4fa2ea6)